src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -142,62 +152,68 @@ ReportServerSideStats(
 
   // Non-ensemble model
   if (stats.composing_models_stat.empty()) {
-    const uint64_t queue_avg_us = AverageDurationInUs(stats.queue_time_ns, cnt);
-    const uint64_t compute_input_avg_us =
-        AverageDurationInUs(stats.compute_input_time_ns, infer_cnt);
-    const uint64_t compute_infer_avg_us =
-        AverageDurationInUs(stats.compute_infer_time_ns, infer_cnt);
-    const uint64_t compute_output_avg_us =
-        AverageDurationInUs(stats.compute_output_time_ns, infer_cnt);
-    const uint64_t compute_avg_us =
-        compute_input_avg_us + compute_infer_avg_us + compute_output_avg_us;
-    const uint64_t cache_hit_avg_us =
+    const uint64_t queue_time_avg_us =
+        AverageDurationInUs(stats.queue_time_ns, stats.queue_count);
+    const uint64_t compute_input_time_avg_us = AverageDurationInUs(
+        stats.compute_input_time_ns, stats.compute_input_count);
+    const uint64_t compute_infer_time_avg_us = AverageDurationInUs(
+        stats.compute_infer_time_ns, stats.compute_infer_count);
+    const uint64_t compute_output_time_avg_us = AverageDurationInUs(
+        stats.compute_output_time_ns, stats.compute_output_count);
+    const uint64_t compute_time_avg_us = compute_input_time_avg_us +
+                                         compute_infer_time_avg_us +
+                                         compute_output_time_avg_us;
+    const uint64_t cache_hit_time_avg_us =
         AverageDurationInUs(stats.cache_hit_time_ns, cache_hit_cnt);
-    const uint64_t cache_miss_avg_us =
+    const uint64_t cache_miss_time_avg_us =
         AverageDurationInUs(stats.cache_miss_time_ns, cache_miss_cnt);
     const uint64_t total_compute_time_ns = stats.compute_input_time_ns +
```

Comment:
This variable is unused variable after the changes

Reply:
Removed 👍